runtime/doc/plugins.txt

```diff
@@ -37,6 +37,91 @@ Help-link		Loaded	Short description
     |pi_spec.txt|	Yes	Filetype plugin to work with rpm spec files
 |tohtml|		Yes	Convert buffer to html, syntax included
 
+==============================================================================
+Builtin plugin: editorconfig                                    *editorconfig*
```

Comment:
Moved editorconfig to plugins.txt.

We can always revisit if plugins.txt gets "too big", but for now it's rather unusual for editorconfig.txt to have its own dedicated helpfile.

It helps to have plugins living in one common area, because it signals to users the mechanisms for controlling them, which are typically driven by keymaps and autocmds rather than builtin options.